make transfer less of a requirement (#864)
ropensci/dev_guide: softwarereview_editor.Rmd

  package, contact [an rOpenSci staff member](https://ropensci.org/about/#team) so they might contact the authors
  about transfer to [the `ropensci-books` GitHub organisation](https://github.com/orgs/ropensci-books).

> #### If the author want their repository to remain under its current owner

Reviewer: ```suggestion
#### Packages remaining in original GitHub organizations

For package authors who wish to retain their repositories in their original GitHub organizations, rather than transfer to github.com/ropensci, editors should:
```

> - Ask them to make a pull request to the [JSON file](https://github.com/ropensci/roregistry/blob/gh-pages/info/not_transferred.json) that lists all repositories that have not been transferred. [Example commit](https://github.com/ropensci/roregistry/commit/9568a77c2081f89d58406efcc9089ef83d09fe59).

Reviewer: ```suggestion
- Ask package authors to make a pull request to the [JSON file](https://github.com/ropensci/roregistry/blob/gh-pages/info/not_transferred.json) that lists all repositories that have not been transferred. [Example commit](https://github.com/ropensci/roregistry/commit/9568a77c2081f89d58406efcc9089ef83d09fe59).
```

> - Ask them to replace the content of the current code of conduct of the repository with the content of the [default code of conduct of the GitHub organization](https://github.com/ropensci/.github/blob/main/CODE_OF_CONDUCT.md) (which means linking to rOpenSci's code of conduct).

Reviewer: ```suggestion
- Ask package to replace the content of the current code of conduct of the repository with the content of the [default code of conduct of the rOpenSci GitHub organization](https://github.com/ropensci/.github/blob/main/CODE_OF_CONDUCT.md).
```

  package, contact [an rOpenSci staff member](https://ropensci.org/about/#team) so they might contact the authors
  about transfer to [the `ropensci-books` GitHub organisation](https://github.com/orgs/ropensci-books).

> #### Packages remaining in original GitHub organizations

Author: For PT translation https://github.com/ropensci/dev_guide/pull/855/files#r2018730631

Reviewer 2: Hi @maelle ! I checked and it's on main:

 https://github.com/ropensci/dev_guide/blob/4260c92c9e23d8eb764e0dc59ae8990f3dc82f51/softwarereview_editor.pt.Rmd#L114-L120

Author: Awesome, thank you!
